### PR TITLE
[TF] To-Do image  resize in image-classification-dataset.md

### PR DESCRIPTION
**Pull request by abhinavsp0730**
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

File: chapter_linear-networks/image-classification-dataset.md [hunk 1]
```diff
@@ -280,12 +280,21 @@ def load_data_fashion_mnist(batch_size, resize=None):  #@save
 #@tab tensorflow
 def load_data_fashion_mnist(batch_size, resize=None):  #@save
     """Download the Fashion-MNIST dataset and then load into memory."""
-    # TODO: Resize
     (mnist_train_x, mnist_train_y), (mnist_test_x, mnist_test_y) = tf.keras.datasets.fashion_mnist.load_data()
-    return (
+    if(resize is None):
```
Comment (terrytangyuan) [inline chapter_linear-networks/image-classification-dataset.md:284]: `if resize is None`

File: chapter_linear-networks/image-classification-dataset.md [hunk 2]
```diff
@@ -280,12 +280,21 @@ def load_data_fashion_mnist(batch_size, resize=None):  #@save
...
+      return (
         tf.data.Dataset.from_tensor_slices(
             (mnist_train_x, mnist_train_y)).batch(batch_size).shuffle(len(mnist_train_x)),
         tf.data.Dataset.from_tensor_slices((mnist_test_x, mnist_test_y)).batch(batch_size))
+    else:
+      def map_fn(img,label):
+        img = tf.reshape(img, (28,28,1))
+        img =  tf.image.resize(img, (resize))
+        return (img,label)
```
Comment (terrytangyuan) [inline chapter_linear-networks/image-classification-dataset.md:293]: `return img, label`

File: chapter_linear-networks/image-classification-dataset.md [hunk 3]
```diff
@@ -280,12 +280,21 @@ def load_data_fashion_mnist(batch_size, resize=None):  #@save
...
     (mnist_train_x, mnist_train_y), (mnist_test_x, mnist_test_y) = tf.keras.datasets.fashion_mnist.load_data()
-    return (
+    if(resize is None):
+      return (
         tf.data.Dataset.from_tensor_slices(
             (mnist_train_x, mnist_train_y)).batch(batch_size).shuffle(len(mnist_train_x)),
         tf.data.Dataset.from_tensor_slices((mnist_test_x, mnist_test_y)).batch(batch_size))
+    else:
+      def map_fn(img,label):
+        img = tf.reshape(img, (28,28,1))
```
Comment (terrytangyuan) [inline chapter_linear-networks/image-classification-dataset.md:291]: space after commas

File: chapter_linear-networks/image-classification-dataset.md [hunk 4]
```diff
@@ -280,12 +280,21 @@ def load_data_fashion_mnist(batch_size, resize=None):  #@save
...
+        img =  tf.image.resize(img, (resize))
+        return (img,label)
+      return (
+        tf.data.Dataset.from_tensor_slices(
```
Comment (terrytangyuan) [inline chapter_linear-networks/image-classification-dataset.md:295]: Move the common code to above.

File: chapter_linear-networks/image-classification-dataset.md [hunk 5]
```diff
@@ -280,12 +280,21 @@ def load_data_fashion_mnist(batch_size, resize=None):  #@save
 #@tab tensorflow
 def load_data_fashion_mnist(batch_size, resize=None):  #@save
```
Comment (terrytangyuan) [inline chapter_linear-networks/image-classification-dataset.md:281]: You need to generate this function in `d2l/tensorflow.py`

File: chapter_linear-networks/image-classification-dataset.md (hunk 4 above)
Comment (abhinavsp0730) [reply chapter_linear-networks/image-classification-dataset.md:295]: I'm not getting what is the common code, here.

**Comment by terrytangyuan** [comment]
This has been fixed. Please try the latest notebooks. Thanks for the attempt!
